### PR TITLE
message: Implement Options.LocationPath() & Options.SetLocationPath()

### PR DESCRIPTION
**Pull request by maribu**
As trivial as the title says

### CODE REVIEW

File: message/options.go [hunk 1]
```diff
@@ -107,10 +107,27 @@ func (options Options) path(buf []byte) (int, error) {
...
+		return "", err
+	}
+	buf = buf[:m]
+	return string(buf), nil
+}
+
+// LocationPath joins Location-Path options by '/' to the buf.
+//
+// Return's number of used buf bytes or error when occurs.
+func (options Options) LocationPath() (string, error) {
```
Comment (Danielius1922) [inline message/options.go:125]: Don't we need a setter or why do only getter is added? 

If I'm reading [rfc7252](https://www.rfc-editor.org/rfc/rfc7252) correctly, LocationPath should be contained in a POST response. If somebody uses go-coap to construct the response we should provide a convenient way to append the option to the message.

Comment (jkralik) [reply message/options.go:139]: Yes. Thx

**Review by jkralik — APPROVED** [review]

**Comment by jkralik** [comment]
Please, could you add a test? thx

**Comment by maribu** [comment]
Done. Shall I squash?

**Comment by jkralik** [comment]
> Done. Shall I squash?

Yes. Thx

File: message/options.go (hunk 1 above)
Comment (maribu) [reply message/options.go:139]: Sorry for asking again: Yes = squash, or yes = keep them separate?

Comment (Danielius1922) [reply message/options.go:139]: Squash it, Jozef wants to squash everything all the time so I'm 100% certain that's what he meant. :D

**Review by Danielius1922 — APPROVED** [review]

**Comment by Danielius1922** [comment]
Interesting, we don't have a SonarCloud or codecov reports, I guess the workflows are not setup properly for forked repositories. Note taken.

**Pull request merged**